### PR TITLE
fix: ESlint errors in graphql-toolbox package

### PR DESCRIPTION
**Pull request by tbwiss**
Addressing all ESlint errors for the `graphql-toolbox` package.

### CODE REVIEW

File: packages/graphql-toolbox/src/modules/EditorView/docexplorer/SearchResults.tsx [hunk 1]
```diff
@@ -5,160 +5,148 @@
  *  LICENSE file in the root directory of this source tree.
  */
 
-import React, { ReactNode } from 'react';
-import { GraphQLSchema, GraphQLNamedType } from 'graphql';
+import React, { ReactNode } from "react";
```
Comment (tbwiss) [inline packages/graphql-toolbox/src/modules/EditorView/docexplorer/SearchResults.tsx:8]: This file/component is from an external source, almost exclusively prettier updates.

File: packages/graphql-toolbox/src/modules/EditorView/docexplorer/TypeDoc.tsx [hunk 2]
```diff
@@ -5,256 +5,236 @@
  *  LICENSE file in the root directory of this source tree.
  */
 
-import React, { ReactNode } from 'react';
+import React, { ReactNode } from "react";
```
Comment (tbwiss) [inline packages/graphql-toolbox/src/modules/EditorView/docexplorer/TypeDoc.tsx:8]: This file/component is from an external source, almost exclusively prettier updates.

File: packages/graphql-toolbox/src/modules/EditorView/docexplorer/TypeLink.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@
  */
 
 import { GraphQLList, GraphQLNonNull, GraphQLType, GraphQLNamedType } from "graphql";
+import { Fragment } from "react";
```
Comment (tbwiss) [inline packages/graphql-toolbox/src/modules/EditorView/docexplorer/TypeLink.tsx:9]: This file/component is from an external source, almost exclusively prettier updates.

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at
 http://neo4j-graphql-toolbox-stage-2259.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-2259.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

File: packages/graphql-toolbox/src/modules/HelpDrawer/Keybindings.tsx [hunk 4]
```diff
@@ -80,15 +80,24 @@ export const Keybindings = ({ onClickClose, onClickBack }: Props): JSX.Element =
     return (
         <div data-test-help-drawer-keybindings-list>
             <div className="flex items-center justify-between">
+                {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */}
```
Comment (mjfwebb) [inline packages/graphql-toolbox/src/modules/HelpDrawer/Keybindings.tsx:83]: Can we make this something like `<button></button>` so we are more a11y friendly?

File: packages/graphql-toolbox/src/contexts/auth.tsx [hunk 5]
```diff
@@ -170,5 +133,41 @@ export function AuthProvider(props: any) {
...
+        } catch (err) {
+            setValue((values) => ({ ...values, isConnected: false }));
+        }
+    };
+
+    const processLoginPayload = (value: State | undefined, loginPayloadFromDesktop: LoginPayload | null) => {
+        let loginPayload: LoginPayload | null = null;
+        if (loginPayloadFromDesktop) {
+            loginPayload = loginPayloadFromDesktop;
+            setValue((v) => ({ ...v, isNeo4jDesktop: true }));
```
Comment (mjfwebb) [inline packages/graphql-toolbox/src/contexts/auth.tsx:150]:
```suggestion
            setValue((values) => ({ ...values, isNeo4jDesktop: true }));
```

Just to keep consistent with other usages of setValue.

**Review by mjfwebb — CHANGES_REQUESTED** [review]

**Review by mjfwebb — DISMISSED** [review]
Looks good!

**Pull request merged**